Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -91,16 +91,8 @@ class NetworkResponseHandler {
 
         // Attempt to decode as an `EdgeResponse` first
         guard let edgeErrorResponse = try? JSONDecoder().decode(EdgeResponse.self, from: data) else {
-
-            // If decoding as an `EdgeResponse` fails, attempt to decode as a generic `EdgeEventError`
-            if let edgeErrorResponse = try? JSONDecoder().decode(EdgeEventError.self, from: data) {
-                Log.debug(label: LOG_TAG, "processResponseOnError - Processing server error response:\n \(jsonError), request id \(requestId)")
-                dispatchEventErrors(errorsArray: [edgeErrorResponse], requestId: requestId)
-            } else {
-                Log.warning(label: LOG_TAG,
-                            "processResponseOnError - The conversion to JSON failed for server error response: \(jsonError), request id \(requestId)")
-            }
-
+            Log.warning(label: LOG_TAG,
+                        "processResponseOnError - The conversion to JSON failed for server error response: \(jsonError), request id \(requestId)")
             return
```

Comment:
It looks like we account for generic errors in lines L107->L115, so we should first attempt to decode to EdgeResponse, then EdgeEventError, without returning here.